gatsby_fp/src/components/fieldComponents/genInfoTable.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+
+export default function GenInfoTable
+({school, title, imgsrc, building, room, phone, fax, username, website}){
+
+  return(
+    <>
+    <div className="table_wrapper">
+          <div className="table_wrapper_inner">
+            <table
+              cellPadding="5"
+              cellSpacing="0"
+              border="0"
+              className="bioregion"
+              width="100%"
+            >
+              <tbody>
+                <tr>
+                  <td valign="top" align="left" width="160px">
+                    <img
+                      src={`data:image/jpeg;base64,${imgsrc}`}
+                      className="bioimage"
+                      style={{ maxWidth: "150px" }}
```

Comment:
Generally speaking, we should avoid using the `style` attribute for styling. CSS works under a model of specificity, where inline styling like this always takes precedence. If we ever wanted to change the `max-width`, we would be unable to do so in any external CSS we write.

A workaround would be to give the `img` a `className`, and style that.